.drone.yml

```diff
@@ -44,8 +44,16 @@ pipeline:
       - php occ app:list
       - php occ app:enable files_antivirus
       - php occ app:list
+      - php occ config:system:set trusted_domains 2 --value=server
```

Comment:
why `2`?

Reply:
Cut-paste from elsewhere. I think 1 is "localhost" by default, and I am not even sure if this is zero-based. I  will change it to 1.